main/main.c

```diff
@@ -68,6 +71,12 @@ static void init_system_tasks();
 void vAssertCalled(unsigned long ulLine, const char *const pcFileName);
 
 int ex2_main(int argc, char **argv) {
+#if DEV_RUN_TEST_SUITE == 1
+  run_test_suite();
```

Comment:
Would this work for the functions that need FreeRTOS tasks? Like, can we call it after `vTaskStartScheduler` ?

Reply:
I plan on discussing this with Andrew and Ron!